src/kinetics/InterfaceKinetics.cpp

```diff
@@ -595,4 +595,30 @@ void InterfaceKinetics::setPhaseStability(const size_t iphase, const int isStabl
     }
 }
 
+double InterfaceKinetics::InterfaceCurrent(int iPhase)
+{
+    int sp = thermo(iPhase).nSpecies();
```

Comment:
This is nitpicky, but I'd recommend something like `nSp` or `n_sp`.  `sp` in this class typically refers to an actual species object.

Reply:
Agreed, I changed to nSp.